### ISSUE TITLE
Tailwindcss-themer does not work with next/font

### ISSUE THREAD

**Issue by ru-van-urk** [RyanClementsHax/tailwindcss-themer#58]
### Describe the bug

Tailwindcss-theme does not work in combination with next/font. This because the css variables get messed up.

When inspecting the variables in the browser I found that the incorrect variables was picked up by the plugin. Not the variable specified by the next/font package.

### Your minimal, reproducible example

https://github.com/ru-van-urk/tailwind-themer-font

### Steps to reproduce

1. Clone repo
2. Start next app `yarn dev`
3. See that the font is gone on the home page
4. Go back to tailwind config and uncomment font-family in the main extend
5. See it working

### Expected behavior

I expect the tailwindcss-themer to behave the same as the normal tailwindcss config

### How often does this bug happen?

Every time

### Platform

-Browser: Firefox, Chrome

### tailwindcss-themer version

3.0.1

**Comment by RyanClementsHax** [comment]
Thanks for opening this. I'll take a look at it when I get the chance.

**Comment by RyanClementsHax** [comment]
Thanks so much for the repo! It was super helpful.

I was able to get everything working by applying the `my-theme` class to the `main` tag in `_app.tsx`. Does this not work for you?

Given that there isn't a default value provided in `defaultTheme`, no css will be applied until `my-theme` is applied. This is expected behavior of the plugin.

Does this help or am I misunderstanding what your issue is?

I'm closing this assuming this can be fixed by applying a default value, but I'll happily reopen it if I'm wrong.

**Comment by ru-van-urk** [comment]
It has been awhile since I looked at this but I did found the problem. See tailwind themer sets the variables on the root. When using next/font the font variables get set on main. So when tailwind themer references them in the root they dont exsist at that level. I solved it by setting a theme ( "my-theme', what you also suggested) on main and never define default values. Works for me but is a bit confusing when setting it up. Once again I havent looked into this since posting the original message but I plan on picking it up later.
